web/cypress/test/quay-db-data.txt

```diff
@@ -6513,8 +6515,8 @@ COPY public.quayservice (id, name) FROM stdin;
 --
 
 COPY public.queueitem (id, queue_name, body, available_after, available, processing_expires, retries_remaining, state_id) FROM stdin;
-1	namespacegc/2/	{"marker_id": 1, "original_username": "quay"}	2023-11-07 19:57:36.102126	t	2023-11-07 22:52:36.037751	5	e90498e4-d92b-487a-90a3-1ab34388af7c
```

Comment:
I usually try to leave out the changes that add the log and update the queue item table as their not relevant

Reply:
ok, reverted these changes